.buildkite/pull_request_pipeline.yml

```diff
@@ -123,6 +123,25 @@ steps:
       manual:
         allowed: true
 
+  - label: "Observability SRE container smoke test"
+    key: "observability-sre-container-smoke-test"
+    agents:
+      provider: gcp
+      imageProject: elastic-images-prod
+      image: family/platform-ingest-logstash-ubuntu-2204
+      machineType: "n2-standard-4"
+      diskSizeGb: 64
+    retry:
+      automatic:
+        - limit: 3
+    command: |
+      set -euo pipefail
+      source .buildkite/scripts/common/vm-agent.sh
+      ./gradlew --stacktrace artifactDockerObservabilitySRE
+      logstash_version="$(awk -F':' '{ if ("logstash" == $1) { gsub(/ /,"",$2); print $2; exit } }' versions.yml)-SNAPSHOT"
```

Comment:
it looks like buildkite interpolates this, and doesn't like our `$1`

Reply:
yeah, i've struggled enough with buildkite intperolation stuff to want to just extract this to a shell script. Updated.